railties/lib/rails/generators/rails/app/templates/config/environments/production.rb.tt

```diff
@@ -67,7 +67,7 @@ Rails.application.configure do
   # Prepend all log lines with the following tags.
   config.log_tags = [ :request_id ]
 
-  # Info include generic and useful information about system operation, but avoids logging too much
+  # Info includes generic and useful information about system operation, but avoids logging too much
```

Comment:
While we are fixing this typo, would you mind also changing `Info` to `"info"`?

```suggestion
  # "info" includes generic and useful information about system operation, but avoids logging too much
```

I think that is more clear and fits better with `If you want to log everything, set the level to "debug".`

Reply:
Yep, sure.